src/screens/login/HSwitchLoginFlow/HyperSwitchEmailVerifyScreen.res

```diff
@@ -50,7 +42,7 @@ let make = (~setAuthType, ~setAuthStatus) => {
   }
 
   React.useEffect0(() => {
-    let body = generateBody(url)
+    let body = HyperSwitchAuthUtils.generateBody(url)
```

Comment:
generateBody of what type?
add the full name for whatever it is accept invite or any other it is used. If moving to utils else it's used only in this file

Reply:
Renamed the function to `generateBodyForEmailRedirection` and moved to util file as it is used in both Email verify screen and Accept invite from email screen